rasa/core/policies/rule_policy.py

```diff
@@ -395,6 +533,8 @@ def train(
         rule_trackers = [t for t in training_trackers if t.is_rule_tracker]
         if self._restrict_rules:
             self._check_rule_restriction(rule_trackers)
+        if self._check_for_contradictions:
+            self._check_rule_action_fingerprints(rule_trackers, domain)
```

Comment:
Why not calling it like this
```suggestion
            self._check_for_incomplete_rules(rule_trackers, domain)
```

Reply:
because I came up with `incomplete` term afterwards 😅